Source/Core/DolphinQt/GrandSettingsWindow.cpp

```diff
@@ -0,0 +1,180 @@
+// Copyright 2020 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <QDialogButtonBox>
+#include <QGridLayout>
+#include <QLineEdit>
+#include <QSortFilterProxyModel>
+#include <QStackedWidget>
+#include <QStringLiteral>
+#include <QTreeWidget>
+
+#include "Common/Config/Config.h"
+#include "Core/Config/MainSettings.h"
+#include "Core/ConfigManager.h"
+
+#include "DolphinQt/Config/CommonControllersWidget.h"
+#include "DolphinQt/Config/GamecubeControllersWidget.h"
+#include "DolphinQt/Config/Graphics/AdvancedWidget.h"
+#include "DolphinQt/Config/Graphics/EnhancementsWidget.h"
+#include "DolphinQt/Config/Graphics/GeneralWidget.h"
+#include "DolphinQt/Config/Graphics/HacksWidget.h"
+#include "DolphinQt/Config/Graphics/SoftwareRendererWidget.h"
+#include "DolphinQt/Config/WiimoteControllersWidget.h"
+#include "DolphinQt/GrandSettingsWindow.h"
+#include "DolphinQt/Settings/AdvancedPane.h"
+#include "DolphinQt/Settings/AudioPane.h"
+#include "DolphinQt/Settings/GameCubePane.h"
+#include "DolphinQt/Settings/GeneralPane.h"
+#include "DolphinQt/Settings/InterfacePane.h"
+#include "DolphinQt/Settings/PathPane.h"
+#include "DolphinQt/Settings/WiiPane.h"
+
+#include "VideoCommon/VideoBackendBase.h"
+#include "VideoCommon/VideoConfig.h"
+
+GrandSettingsWindow::GrandSettingsWindow(X11Utils::XRRConfiguration* xrr_config, QWidget* parent)
+    : GraphicsDialog(parent), m_xrr_config(xrr_config)
+{
+  // Set Window Properties
+  setWindowTitle(tr("Grand Settings"));
+  setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
+
+  m_search_bar = new QLineEdit();
+  m_search_bar->setPlaceholderText(QStringLiteral("Search settings"));
```

Comment:
All of these `QStringLiteral` usages should be replaced with `tr` so they get translated.

Reply:
I tend to forget about the `tr` thing.  I'll update this, thanks Techjar!